docs/tutorials/intro.mdx

```diff
@@ -2,11 +2,88 @@
 sidebar_position: 1
 ---
 
-# Velaptor Intro
+# Intro to Velaptor
 
-<div style={{textAlign: 'center'}}>
+Velaptor is an open-source framework for building high-performant, high-quality, cross-platform 2D games using C#/dotnet.
 
-<img style={{textAlign: 'center', width: '25%'}} src="/img/velaptor.png"/>
+:::info
+[Here](https://www.freecodecamp.org/news/the-difference-between-a-framework-and-a-library-bd133054023f/) is a good explanation of a framework vs a library.
+:::
 
-</div>
+## Overview
 
+Velaptor focuses on developing your game.  What this means is that instead of dealing with low-level details such as writing low level
+code for rendering graphics, keyboard and mouse input, sound, and other common things required by games, you can just focus on writing your game.
+
+Writing these things from scratch can take hours when all you want to do is create.  This is where Velaptor can help!!
```

Comment:
Grammar
```suggestion
Velaptor focuses on creating your game instead of building time-consuming details.  Writing low level code for rendering graphics, keyboard and mouse input, sound, and other common things required by games can take hours (trust me, I know).  With Velaptor, you can just focus on writing your game!!
```

Reply:
I like it.